packages/core/useMutationObserver/index.ts

```diff
@@ -56,6 +60,7 @@ export function useMutationObserver(
   return {
     isSupported,
     stop,
+    takeRecords,
```

Comment:
Maybe just return the `observer` variables.
```suggestion
    observer,
```

Reply:
If we return `observer`, we need to wrap it as a `Ref`.

Reply:
I think make it to `shallowRef` is more flexible

Reply:
Both are possible. Vue only proxies native `Object` and does not proxy `MutationObserver`, for `MutationObserver`, the behavior is consistent between `Ref` and `shallowRef `.

Reply:
However, I believe we should return a `takeRecords` function. Exposing the `observer` to the user would allow them to manually call `disconnect` to stop the listening, but we prefer users to use `stop` to halt the listening. This is because `stop` can also stop the `watch`. In the `MutationObserver` instance, only the `observer`, `takeRecords`, and `disconnect` methods are available. The `observer` is automatically executed, and we expose the `disconnect` through `stop` to the user. We only need to expose the `takeRecords` to the user.